backup.go

```diff
@@ -72,7 +73,7 @@ func (b *BackupServiceHandler) List(ctx context.Context, options *ListOptions) (
 	req.URL.RawQuery = newValues.Encode()
 
 	backups := new(backupsBase)
-	if err = b.client.DoWithContext(ctx, req, backups); err != nil {
+	if _, err = b.client.DoWithContext(ctx, req, backups); err != nil {
```

Comment:
Shouldn't this function be returning the API json response as well? Backup -> List()

Reply:
@christhemorse That is correct. all set.